netebpfext/user/fwp_um.cpp

```diff
@@ -476,13 +488,21 @@ _IRQL_requires_max_(PASSIVE_LEVEL) NTSTATUS FwpmTransactionCommit0(_In_ _Release
 
 _IRQL_requires_max_(PASSIVE_LEVEL) NTSTATUS FwpmTransactionAbort0(_In_ _Releases_lock_(_Curr_) HANDLE engine_handle)
 {
+    if (ebpf_fault_injection_inject_fault()) {
+        return STATUS_NO_MEMORY;
```

Comment:
Check with WFP folks if it makes sense to fail abort with OOM.

Reply:
The existing code checks the return status from FwpmTransactionAbort0, and adds a trace. So, followed the same pattern of adding trace.

In net_ebpf_ext.c,
if (is_in_transaction) {
            status = FwpmTransactionAbort(_fwp_engine_handle);
            if (!NT_SUCCESS(status)) {
                NET_EBPF_EXT_LOG_NTSTATUS_API_FAILURE(
                    NET_EBPF_EXT_TRACELOG_KEYWORD_ERROR, "FwpmTransactionAbort", status);
            }
        }